### ISSUE TITLE
docker: no matching manifest for linux/arm64/v8 in the manifest list entries.

### ISSUE THREAD

**Issue by sandeepraghu** [osalvador/ReplicaDB#171]
sandeepraghuvanshi@Sandeeps-MacBook-Pro ReplicaDB % docker run \
    -v /tmp/replicadb.conf:/home/replicadb/conf/replicadb.conf \
    osalvador/replicadb
Unable to find image 'osalvador/replicadb:latest' locally
latest: Pulling from osalvador/replicadb
docker: no matching manifest for linux/arm64/v8 in the manifest list entries.
See 'docker run --help'.

**Comment by MichalisDBA** [comment]
@sandeepraghu Hi. I think replicadb does not have a docker image for arm64/v8 published.

**Comment by sandeepraghu** [comment]
@MichalisDBA , yes you are correct, i dont found any tags for this , then what can be done to move ahead please suggest.
